Makefile.work

```diff
@@ -51,12 +51,14 @@ comma := ,
 
 ifeq ($(shell uname -m), aarch64)
 COMPILE_HOST_ARCH := arm64
-else ifeq ($(shell uname -m), armv7l)
+else ifneq (,$(filter $(shell uname -m),armv7l armv8l))
```

Comment:
>ifneq [](http://example.com/codeflow?start=5&length=5)

Is it possible to test postively instead of negatively?

Reply:
Changed to use ifeq, thanks.